src/features/import/l10n/messageIds.ts

```diff
@@ -285,6 +285,12 @@ export default makeMessages('feat.import', {
         ),
         title: m('You have not configured identifying columns'),
       },
+      unexpectedError: {
+        description: m(
+          'No people have been imported. You can go back and check the import settings or select a new file to import. There were errors in the form you submitted. Please try again and make sure you fill in all the necessary information. If the error persist you can contact support at info@zetkin.org'
```

Comment:
Thinking a bit about this text... I think it could be a bit shorter. my suggestion: 

"Please go back and check the settings you've made, or select a new file to import. If the error persist you can contact support at info@zetkin.org"